### DISCUSSION TITLE
February 2024 Comm Call

### DISCUSSION THREAD

**Discussion by ikrishg** [EddieHubCommunity/support#8699] [Community Call]
hiya EddieHubbers :wave: 

This is a gentle reminder of @public speaking session going to be held this on 25th Feb.

Everyone who wants to improve their public speaking and kick start their public speaking journey can join this session. You are free to do the following things in the session:

1. Introduce yourself in brief and know your peers.
2. Share your knowledge and experience.
3. Involve in the discussion on a specific topic
4. Provide & get suggestions to help improve yourself and your peers.

Time: 20.30 IST 
Place:  #"RadioEddieStage" 
Topic:  Introductions + Water cooler

Feel free to join us. Ask questions if you have any doubts. These sessions are for everyone to learn. Kindly read the code of conduct before attending. Feel free to give suggestions for future topics you would love to discuss on.

Ref: <https://discord.com/events/699608417039286293/1206219882371481610>

**Comment by ikrishg** [comment]
Feel free to drop your introductions / biodrops here 👇🏻 (Also please mention your twitter so that we can tag you)

**Reply by Joramzy** [reply]
X @Joramjohnmai

**Reply by ashutosh7i** [reply]
i made my biodrop- https://www.biodrop.io/ashutosh7i

**Reply by inclinedadarsh** [reply]
Heyy heyy everyone!
I'm Adarsh Dubey, a web developer, ml engineer and community builder.

Great to see you all here.
Here's my twitter: https://twitter.com/inclinedadarsh
Here's my portfolio: https://adarshdubey.com/

**Reply by Nishantjain10** [reply]
This is Nishant from India. I am frontend dev, with some experience in backend engineering. I like contributing in open source and participating in hackathons is something I enjoy.🌻

Here's my portfolio website: https://devnishant.vercel.app/ (bit outdated, revamping xd)
Here's my biodrop: https://www.biodrop.io/Nishantjain10

let's connect:
Twitter: https://twitter.com/devnishant10
Linkedin: https://www.linkedin.com/in/nishantj2002
Github: https://github.com/nishantjain10

**Comment by HeraclesG** [comment]
Hello everyone. I respect all of you and want to a lot from you. thank you.
